tests/specs/benchmarks/verification.k

```diff
@@ -249,9 +249,11 @@ module VERIFICATION
 
     rule 0 <Int 1 <<Int N => true requires 0 <=Int N [simplification]
 
-    rule X <=Int #ceil32(X) => true requires X >=Int 0 [simplification]
+    rule X +Int Y <Int pow256 => true requires X <=Int pow16 andBool Y <Int pow16 [simplification]
 
-    rule #ceil32(N) => ((N +Int 31) /Int 32) *Int 32 [smt-lemma]
+    rule X <Int #ceil32(X)      => true  requires 0 <=Int X                       [simplification]
```

Comment:
I believe this is `X <=Int #ceil32(X)      => true`

Reply:
Fixed.